src/Microsoft.AspNet.Authentication/Events/SigningInContext.cs

```diff
@@ -25,6 +28,13 @@ public abstract class ReturnEndpointContext : EndpointContext
         public ClaimsPrincipal Principal { get; set; }
         public AuthenticationProperties Properties { get; set; }
 
+        public bool IsRequestCompleted { get; private set; }
+
+        public void RequestCompleted()
+        {
+            IsRequestCompleted = true;
```

Comment:
You can't remove a private set unless its the ctor that's setting it.  No private set, caused a compile error when trying to set it in the method...